.github/workflows/test.yml

```diff
@@ -152,3 +152,15 @@ jobs:
             - uses: Swatinem/rust-cache@v2
             - name: forge fmt
               run: cargo run --bin forge -- fmt --check testdata/
+
+    feature-checks:
```

Comment:
wonder how much time this will add to ci

Reply:
should complete before tests if it runs in parallel with them